docs/core/porting/upgrade-assistant-telemetry.md

```diff
@@ -12,14 +12,70 @@ The [Upgrade Assistant](./upgrade-assistant-overview.md) includes a telemetry fe
 
 The Upgrade Assistant telemetry feature is enabled by default. To opt out of the telemetry feature, set the `DOTNET_UPGRADEASSISTANT_TELEMETRY_OPTOUT` environment variable to `1` or `true`.
 
+### [Command Line](#tab/command-line)
```

Comment:
Is the correct? Command line? Not "console?"

Reply:
```suggestion
### [Console](#tab/console)
```